liblangutil/SourceReferenceFormatter.h

```diff
@@ -51,7 +52,17 @@ class SourceReferenceFormatter
 	virtual void printExceptionInformation(SourceReferenceExtractor::Message const& _msg);
 
 	virtual void printSourceLocation(SourceLocation const* _location);
-	virtual void printExceptionInformation(dev::Exception const& _error, std::string const& _category);
+	virtual void printExceptionInformation(dev::Exception const& _exception, std::string const& _category);
+
+	/// When we don't care detailed type of error, we can use these shortcuts to format Error
+	virtual void printErrorInformation(Error const& _error);
+	static std::string formatErrorInformation(Error const& _error)
```

Comment:
Could you please add a new line before this definition and move the other one line up? With this all virtual declarations are contained in one block.